### ISSUE TITLE
accessibility

### ISSUE THREAD

**Issue by shimi12x** [home-assistant/home-assistant.github.io#31999]
### Feedback

Acessibility for the blind

The home assistant server does not have the option to install a screen reader or to add addons to improve accessibility.

Please consider adding accessibility features to home assistant project

Thanks

### URL

https://www.home-assistant.io/getting-started/

### Version

2024.3.1

**Comment by frenck** [comment]
Hi there @shimi12x 👋 

Thanks for your message. We do have contributors in our members pool that focus on accessibility (one of our frontend developers is legally blind). We take accessibility seriously. 

If you find issues with this, please report them in our frontend repository's issue tracker so they can be looked at. However, I'm going to close this issue, as this is submitted as documentation feedback. There is no amount of documentation we can adjust to improve the accessibility of the Home Assistant software.

../Frenck

**Comment by shimi12x** [comment]
Hi franck

Thanks for the quick response

My qustion is how a blind person can install
The HA server on a VM without requesting help from a seeing person.

Is there an option to receive an audio feedback during the installation process

Thanks
Shimi12x

-------- Original Message --------
